libs/sdk-ui-ext/src/internal/translations/en-US.json

```diff
@@ -576,6 +576,21 @@
         "comment": "",
         "limit": 0
     },
+    "properties.axis.format": {
+        "value": "Format",
+        "comment": "",
+        "limit": 0
+    },
+    "properties.axis.format.inherit": {
+        "value": "inherit",
+        "comment": "Axis format option to inherit formatting from used measure.",
+        "limit": 0
+    },
+    "properties.axis.format.info.inherit": {
+        "value": "The format is inherited from the first measure in the insight.",
```

Comment:
Insight should have also a sibling text with Report 🥲 .

Reply:
Added.